src/fsharp/PrettyNaming.fs

```diff
@@ -131,10 +131,20 @@ module internal Microsoft.FSharp.Compiler.PrettyNaming
         let nameLen = name.Length
         let rec loop i = (i < nameLen && (opCharSet.Contains(name.[i]) || loop (i+1)))
         loop 0
+    
+    /// Returns the plain name of a given identifier, operator or double backticked name, 
+    /// e.g. "|>>" instead of "( |>> )" or "long identifier" instead of "( long identifier )".
+    let DemangleBacktickedName (name:string) =
```

Comment:
I'd need to look more closely.  Something smells here - somehow it seems wrong that the parentheses are present at all for these names - IIRC the parentheses are only added for output purposes - but now we are removing them and adding the backticks.  Something just feels wrong, like we have multiple versions of "display text for identifier" and aren't distinguishing them very well.

In the meantime something like ``DemangleParentheseesForBacktickIdentifier`` or something I suppose.  But the difficulty in naming seems to indicate some other problem.

It's not a big deal, and I've not reviewed this code in a while, just glancing.